incubator/hnc/test/conversion/conversion_test.go

```diff
@@ -272,10 +306,8 @@ func setupV1alpha2(){
 	MustRun("kubectl apply -f ../../manifests/hnc-manager.yaml")
 	// Wait for the cert rotator to write caBundles in CRD conversion webhooks.
 	ensureCRDConvWHReady()
-}
 
-// Verify CRDs still have 'v1alpha1' in spec.versions but not in status.storedVersions.
-func verifyCRDConversion(){
+	// Verify CRDs still have 'v1alpha1' in spec.versions but not in status.storedVersions.
```

Comment:
Maybe still keep the `verifyCRDConversion()` func and put the func call at the end of `setupv1a2`?

Reply:
Why? We only ever do the verification after the upgrade, and we always want to verify after an upgrade.

Reply:
Ah right, sgtm.